### ISSUE TITLE
Recording cant create new file

### ISSUE THREAD

**Issue by Soundofdarkness** [1lann/lol-replay#5]
Hey , after fixing the current spectator url's , i tried using your program for recording games,
but im permantely recieving this error : 
`2018/02/04 14:33:53 failed to read recording TR1_666986296.glr: seek recordings/TR1_666986296.glr: Es wurde versucht, den Dateizeiger vor den Anfang der Datei zu bewegen.`
The german part of it basically translates to : "Failed moving file pointer infront of the beginning of the file"
The Error happens [here](https://github.com/1lann/lol-replay/blob/c7ec366fa41f8b4e70af8d5febcfb2d87cff62af/recording/recording.go#L114)

Edit: OS is windows , and im aware of it not being supported , but im pretty sure thats one of the only parts creating problems for it

**Comment by 1lann** [comment]
Yeah, Windows is not really supported. If you remove the if statements on lines 116 and 117 in recording.go such that on any error, `ErrMissingData` is returned, that will probably fix it.

**Comment by Soundofdarkness** [comment]
Okay im going to try this ^^ Thank you 😄

**Comment by Soundofdarkness** [comment]
Okay , seems like it can record the files correctly now. Only issue im seeing now is that its while replaying just stuck on the last minute of the of the recording , and doesnt render any champ movement, even when scrolling back. Everything is just stuck in one position. Is that maybe a never version of the spectator system ?

It opens like this and thats about it: [ Image](http://whats.eleria.de/files/5c41eb1f8cc5fc02432aedbce8da5e5a9c4e9b5f.jpg)
